config.json

```diff
@@ -31,11 +31,17 @@
       "defaultValue": false
     },
     "RuntimeOS": {
-      "description": "The RuntimeOS used for building and restoring native OS dependent assets (i.e. RID specific binaries)",
+      "description": "The RuntimeOS of the build system.",
       "valueType": "property",
       "values": ["win7", "osx.10.10", "ubuntu.14.04", "ubuntu.16.04", "etc-other-rid-based-os-names"],
       "defaultValue": ""
     },
+    "PackageRID": {
+      "description": "The RID of the target package.",
+      "valueType": "property",
+      "values": ["win7-x64", "ubuntu.14.04-x64"],
+      "defaultValue": "${RuntimeOS}-${ArchGroup}"
```

Comment:
Ah. I assumed this isn't used. The value I put here is the one that gets defaulted by the project. Should I clear it out?